onboard/catkin_ws/src/controls/scripts/drc_utils.py

```diff
@@ -0,0 +1,71 @@
+from tf.transformations import euler_from_quaternion, quaternion_multiply, quaternion_conjugate
+
+
+def get_directions():
```

Comment:
Maybe think of a better word for this than directions, directions implies only xyz. Maybe get_axes?

Reply:
fixed in 189a96e